docs/setup/kubeedge_configure.md

```diff
@@ -0,0 +1,369 @@
+# KubeEdge Configuration
+
+KubeEdge requires configuration on both [Cloud side (KubeEdge Master Node)](#configuration-cloud-side-kubeedge-master-node) and [Edge side (KubeEdge Worker Node)](#configuration-edge-side-kubeedge-worker-node)
+
+## Configuration Cloud side (KubeEdge Master Node)
+
+Setting up cloud side requires two steps 
+
+1. Modification of the configuration files
+2. Adding the edge nodes (KubeEdge Worker Node) on the Cloud side KubeEdge Master Node).
+
+### Modification of the configuration files
+
+Cloudcore requires three configuration file as of now. However this might change in future as per the [Issue 1171](https://github.com/kubeedge/kubeedge/issues/1171)
+
+1. controller.yaml
+2. logging.yaml
+3. modules.yaml
+
+
+Set cloudcore config file
+
+```shell
+    cd ~/cmd/conf
+    vim controller.yaml
+```
+
+Verify the configurations before running `cloudcore`
+    
+```yaml
+controller:
+    kube:
+        master:     # kube-apiserver address (such as:http://localhost:8080)
+        namespace: ""
+        content_type: "application/vnd.kubernetes.protobuf"
+        qps: 5
+        burst: 10
+        node_update_frequency: 10
+        kubeconfig: "/root/.kube/config"   #Enter absolute path to kubeconfig file to enable https connection to k8s apiserver, if master and kubeconfig are both set, master will override any value in kubeconfig.
+    cloudhub:
+      protocol_websocket: true # enable websocket protocol
+      port: 10000 # open port for websocket server
+      protocol_quic: true # enable quic protocol
+      quic_port: 10001 # open prot for quic server
+      max_incomingstreams: 10000 # the max incoming stream for quic server
+      enable_uds: true # enable unix domain socket protocol
+      uds_address: unix:///var/lib/kubeedge/kubeedge.sock # unix domain socket address
+      address: 0.0.0.0
+      ca: /etc/kubeedge/ca/rootCA.crt
+      cert: /etc/kubeedge/certs/edge.crt
+      key: /etc/kubeedge/certs/edge.key
+      keepalive-interval: 30
+      write-timeout: 30
+      node-limit: 10
+    devicecontroller:
+      kube:
+        master:        # kube-apiserver address (such as:http://localhost:8080)
+        namespace: ""
+        content_type: "application/vnd.kubernetes.protobuf"
+        qps: 5
+        burst: 10
+        kubeconfig: "/root/.kube/config" #Enter absolute path to kubeconfig file to enable https connection to k8s apiserver,if master and kubeconfig are both set, master will override any value in kubeconfig.
+```
+
+cloudcore default supports https connection to Kubernetes apiserver, so you need to check whether the path for `controller.kube.kubeconfig` and `devicecontroller.kube.kubeconfig` exist, but if `master` and `kubeconfig` are both set, `master` will override any value in kubeconfig.
```

Comment:
I like a little bit different type of starting the sentence and like others already mentioned the sentence has to break down. I will give you an example perhaps this could help:

By default cloudcore supports https connections to an Kubernetes apiserver. In the configuration file the path `kubeconfig` or  `master` in the sections `controller.kube` and `devicecontroller.kube` has to exist.